PaaConditionalMenu/src/paaconditionalmenu/PaaConditionalMenu.java

```diff
@@ -0,0 +1,86 @@
+package paaconditionalmenu;
+
+import java.util.Scanner;   // Needed for Scanner class
+import java.util.Date;      // Needed for Date and Time
+import java.text.DateFormat;    // Needed for Date and Time
+import java.text.SimpleDateFormat;  // Needed for Date and Time
+import java.util.Calendar;      // Needed for Date and Time
+
```

Comment:
I didn't remember reading about the class header comments, I'll have to go back and read over that part again. Thanks!